yucca/functional/transforms/label_transforms.py

```diff
@@ -13,36 +13,59 @@ def translate_region_labels(regions: dict, labels: dict) -> dict:
     return regions_with_int_labels
 
 
-def convert_labels_to_regions(label, regions):
+def batch_convert_labels_to_regions(label, regions):
     b, c, *shape = label.shape
     assert c == 1, f"Class labels are not onehot encoded. Channel dim must be 1, but got c={c}."
     region_canvas = np.zeros((b, len(regions), *shape))
 
     for channel, region in enumerate(regions.keys()):
         region_labels = regions[region]["labels"]
+        assert isinstance(region_labels[0], int)
         region_canvas[:, channel] = np.isin(label[:, 0], region_labels)
     region_canvas = region_canvas.astype(np.uint8)
 
     return region_canvas
 
 
-# Currently not used, and incorrect with new regions format.
+def convert_labels_to_regions(data, regions):
```

Comment:
Maybe i am not reading this correct, but shouldn't this use the new "priority" property in the region dict?

Reply:
Not quite, priority is something we should look to use when we go the other way around (where it should be added) so we inflate label of shape (1, h, w, d) to (r, h, w, d) in the order of the regions, then when we go back to (1, h, w, d) we want to check that we put the "overlay" the regions in the correct order, so the priority 1 region overwrites any other labels. That however, is not yet implemented which it should be - but it is not necessary for what we do atm, since the brats eval never goes back from regions to labels